decidim-accountability/app/commands/decidim/accountability/admin/update_status.rb

```diff
@@ -5,44 +5,8 @@ module Accountability
     module Admin
       # This command is executed when the user changes a Result from the admin
```

Comment:
```suggestion
      # This command is executed when the user changes a Status from the admin
```

Reply:
done in 94771d78819c3002354367f20f75b28d814444b9